__tests__/hooks/useGroupSetting.test.js

```diff
@@ -284,21 +266,4 @@ describe('use groupSetting should run correctly', () => {
     expect(result.current.base64Profile).toEqual('123');
     expect(result.current.urlImage).toEqual('https://image.jpg');
   });
-
-  it('updateDataEdit should run correctly', async () => {
-    const navigation = {
-      navigate: jest.fn(),
-      push: jest.fn(),
-      goBack: jest.fn()
-    };
-    const route = {
-      params: {
-        username: 'agita'
-      }
-    };
-    const {result} = renderHook(() => useGroupSetting({navigation, route}), {wrapper});
-    await result.current.updateDataEdit('123', true);
-    expect(mockDataEdit).toHaveBeenCalled();
-    expect(navigation.navigate).toHaveBeenCalled();
-  });
 });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case for `updateDataEdit` has been removed. If this function is no longer present or its functionality has been covered elsewhere, this removal is fine. Otherwise, this could lead to less coverage in your tests.
```diff
-  it('updateDataEdit should run correctly', async () => {
-    const navigation = {
-      navigate: jest.fn(),
-      push: jest.fn(),
-      goBack: jest.fn()
-    };
-    const route = {
-      params: {
-        username: 'agita'
-      }
-    };
-    const {result} = renderHook(() => useGroupSetting({navigation, route}), {wrapper});
-    await result.current.updateDataEdit('123', true);
-    expect(mockDataEdit).toHaveBeenCalled();
-    expect(navigation.navigate).toHaveBeenCalled();
-  });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->